### ISSUE TITLE
On typing `A` only first `A` will disappear and then second

### ISSUE THREAD

**Issue by shubham9411** [Kaushal196/typing_game#2]
I think that on typing A only first A will have to disappear and on tapping extra key there will be minus points as well.

**Comment by Kaushal196** [comment]
yes..u are right..but how can we know that if two or more A is on screen who will disappear first on keypress value for all A is same so all a is going to disappear.

**Comment by Kaushal196** [comment]
minus points is nice idea...

**Comment by shubham9411** [comment]
In my last commit i've added this functionality of deleting the top `A` not all. I have done this functionality by getting their position from the top and remove the element nearest to the top. And we can minus the points if user tapped a key and there is no element available.

**Comment by Kaushal196** [comment]
works perfectly..nice job..thanks.
